stack-alpine.yaml

```diff
@@ -0,0 +1,30 @@
+resolver: lts-14.27
```

Comment:
Will we need to remember to keep this in sync with `stack.yaml`?

I think the same adjustments could be achieved by passing arguments to `stack` rather than needing a separate file, but not sure which approach will end up being the bigger maintenance burden.

Reply:
Originally I thought this would work out really nicely, but after fighting with the CI system all day, I still ended up needing to pass in `--system-ghc` and other such things at unexpected places. I like your suggestion here: I'll remove the extra file.